spec/SignIn_SignOut_UserPage/checklist_auth.txt

```diff
@@ -0,0 +1,38 @@
+[‘Main’ page] Verify that User can open ‘Authentication’ page if ‘Sign in' button is clicked .
+
+
+[‘Authentication’ page, ‘Create your Account’ window] Verify that User can open ‘Create your Account’ page if ‘Email address’ field is filled with valid data.
+[‘Authentication’ page, ‘Create your Account’ window] Verify that User can’t open ‘Create your Account’ page if ‘Email address’ field is filled  with invalid data. ‘Invalid email’ message is shown.
+[‘Authentication’ page, ‘Create your Account’ window] Verify that User can’t open ‘Create your Account’ page with empty ‘Email address’ field. ‘Invalid email’ message is shown.
+[‘Authentication’ page, ‘Create your Account’ window] Verify that User can’t open ‘Create your Account’ page if ‘Email address’ field is filled  with email, which already exists in app. ‘An account using this email address has already been registered. Please enter a valid password or request a new one.’ message is shown.
+[‘Authentication’ page, ‘Already Registered?’ window] Verify that User can enter in system if ‘Email address’ and ‘Password’ fields are filled with valid data.‘My Account’ page is opened.
+[‘Authentication’ page, ‘Already Registered?’ window] Verify that User can’t enter in system  if ‘Email address’ and ‘Password’ fields are filled with invalid data.‘Authentication failed’ message is shown.
```

Comment:
Would be nice to add precise checklists with invalid data: empty, invalid format, non-existing, etc.